consensus/ethash/consensus.go

```diff
@@ -180,10 +181,7 @@ func (ethash *Ethash) verifyHeaderWorker(chain consensus.ChainHeaderReader, head
 	if parent == nil {
 		return consensus.ErrUnknownAncestor
 	}
-	if chain.GetHeader(headers[index].Hash(), headers[index].Number.Uint64()) != nil {
```

Comment:
Why remove this check ?

Reply:
See https://github.com/holiman/headerimport/blob/main/README.md#1m-blocks-on-21471